docs/content/developer/cryptography/transaction-auth/offline-signing.mdx

```diff
@@ -18,7 +18,7 @@ You must serialize transaction data following [Binary Canonical Serialization](h
 The following example demonstrates how to serialize data for a transfer using the [IOTA CLI](../../../references/cli.mdx). This returns serialized transaction data in Base64. Submit the raw transaction to execute as `tx_bytes`.
 
 ```shell
-iota client transfer-iota --to <IOTA-ADDRESS> --iota-coin-object-id <COIN-OBJECT-ID> --serialize-unsigned-transaction
+iota client transfer --to <IOTA-ADDRESS> --object-id <COIN-OBJECT-ID> --serialize-unsigned-transaction
```

Comment:
It's not supposed to be `pay-all-iota` here?

Reply:
If it should use the same object as gas coin yes, but doesn't seem necessary for this guide
But better since it doesn't require 2 coins, so I will change it